cmake/Cuda.cmake

```diff
@@ -172,7 +173,7 @@ macro(mshadow_cuda_compile objlist_variable)
     list(APPEND CUDA_NVCC_FLAGS -Xcompiler -Wno-unused-function)
   endif()
 
-  set(CUDA_NVCC_FLAGS_DEBUG "${CUDA_NVCC_FLAGS_DEBUG} -G -lineinfo")
+  set(CUDA_NVCC_FLAGS_DEBUG "${CUDA_NVCC_FLAGS_DEBUG} -G")
```

Comment:
+1 to this.  lineinfo often conflicts with NVCC FLAGs and causes overly verbose warnings.  I'd leave it out.  Those who know what it is know how to use it.